src/pages/TestPage/TestPage.jsx

```diff
@@ -1,20 +1,34 @@
 import "./TestPage.scss";
+<<<<<<< HEAD
+import { useEffect, useState } from "react";
+import Modal from "react-modal";
+import { Button } from "@mui/material";
+import { PopUpModal } from "../../components/PopUpModal/PopUpModal";
+import { getAllJobsData } from "../../utils/Functions/functions";
+=======
 import Button from "@mui/material/Button";
 import { useState } from "react";
 import Modal from "react-modal";
 import { PopUpModal } from "../../components/PopUpModal/PopUpModal";
+>>>>>>> 3c2c5829efbb1817eefd755f8a15c335bce18236
 import LoadingPage from "../LoadingPage/LoadingPage";
 
 const TestPage = () => {
   const [isModalOpen, setModalOpen] = useState(false);
+  const [jobs, setJobs] = useState("");
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
If you are defining the function, you might as well move it out of the useEffect and place it to the top or the bottom of the file. Or just not define it at all